### PR TITLE
Added Schema Validator using base module

### PR DESCRIPTION
**Pull request by jonnyshaw89**
### What is the context of this PR?
This PR adds the eq-schema-validator to the developer setup of EQ

### How to review
Deploy it and test that Schema Validator is functional

### CODE REVIEW

File: survey-runner.tf [hunk 1]
```diff
@@ -106,6 +106,20 @@ module "survey-launcher-for-ecs" {
...
+  env                             = "${var.env}"
+  aws_access_key                  = "${var.aws_access_key}"
+  aws_secret_key                  = "${var.aws_secret_key}"
+  dns_zone_name                   = "${var.dns_zone_name}"
+  ecs_cluster_name                = "${module.eq-ecs.ecs_cluster_name}"
+  aws_alb_listener_arn            = "${module.eq-ecs.aws_alb_listener_arn}"
+  container_name                  = "eq-schema-validator"
+  container_port                  = 5000
+  container_tag                   = "migrate-validation-logic"
+  container_environment_variables = "{\"name\": \"EXAMPLE_VAR\",\"value\": \"TEST\"}"
```
Comment (collisdigital) [inline survey-runner.tf]: Should this be here?

Comment (jonnyshaw89) [reply survey-runner.tf]: Yeh, @ajmaddaford wanted it left as an example.

**Review by MIslam92 — APPROVED** [review]
Ran and received correct status code 👍

**Review by dwyeradam — APPROVED** [review]

**Comment by dwyeradam** [comment]
@jonnyshaw89 Squishy squashy!!

**Comment by MIslam92** [comment]
@jonnyshaw89 "Squishy Squashy" please

**Pull request merged**